src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java

```diff
@@ -46,7 +46,7 @@
  * <h3>Usage examples</h3>
  *
  * <pre>
```

Comment:
The program group should be VariantEvaluationProgramGroup. Please streamline the example command.

Reply:
Fixed